JS/Lesson31/onder/my-app/src/components/createTask/İndex.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import './style.css';
+
+export const CreateTask = ({ onCreateProps }) => {
+
+  const [task, setTask] = useState("");
+
+  const handleInputChange = (event) => {
+    setTask(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (task.length > 0) {
+      onCreateProps(task);
+      setTask("");
+    }
+  };
+
+  return (
+    <>
+      <h2>Create task </h2>
+     
+      <form className="form" onSubmit={handleSubmit}>
```

Comment:
An input with type text is missing in this form so it's not possible to add a new todo right now.
```suggestion
      <form className="form" onSubmit={handleSubmit}>
      <input type="text" onChange={handleInputChange} />
```